.github/workflows/releases-test.yaml

```diff
@@ -0,0 +1,272 @@
+          tag="${TAG:-unknown}"
+          html_url="${HTML_URL:-https://github.com/$repo/releases}"
+          missing_optional="${MISSING_OPTIONAL:-(none listed)}"
+
+          text=":warning: *Release monitor WARNING for \`$repo\`*\n\
+          • *Tag:* \`$tag\`\n\
+          • *Missing OPTIONAL (Windows) assets:* $missing_optional\n\
+          • *Note:* Linux/macOS assets are OK. This is a warning only.\n\
+          • *Release page:* $html_url"
+
+          payload=$(jq -Rn --arg text "$text" '{text: $text}')
+
+          curl -X POST -H 'Content-type: application/json' \
+               --data "$payload" \
+               "$SLACK_PROD_ALERTS_WEBHOOK_URL"
+
+      - name: Log success
+        if: steps.check.outputs.ok == 'true' && steps.check.outputs.missing_optional == ''
+        run: |
+          echo "✅ Release monitor: all required and optional assets are present on latest release."
+
+  test_install_paths:
+    runs-on: ubuntu-24.04
+    needs: check_latest_release
+    if: always()
+    steps:
+      - name: Show latest release info (from monitor)
+        run: |
+          echo "Latest tag from monitor: '${{ needs.check_latest_release.outputs.tag }}'"
+          echo "Release URL from monitor: '${{ needs.check_latest_release.outputs.html_url }}'"
+
+      - name: Test Homebrew install (brew install metalbear-co/mirrord/mirrord)
+        id: brew
+        run: |
+          set -u
+          if ! command -v brew >/dev/null 2>&1; then
+            echo "ok=false" >> "$GITHUB_OUTPUT"
+            echo "error=Homebrew (brew) is not installed on this runner" >> "$GITHUB_OUTPUT"
+            exit 0
+          fi
+
+          # Try to install mirrord via Homebrew tap
+          if ! brew install metalbear-co/mirrord/mirrord >/tmp/brew_install.log 2>&1; then
+            echo "ok=false" >> "$GITHUB_OUTPUT"
+            echo "error=brew install metalbear-co/mirrord/mirrord failed. See logs in this step." >> "$GITHUB_OUTPUT"
+            echo "----- brew output -----"
+            cat /tmp/brew_install.log || true
+            exit 0
+          fi
+
+          echo "ok=true" >> "$GITHUB_OUTPUT"
+          echo "error=" >> "$GITHUB_OUTPUT"
+          echo "brew install succeeded."
+
+      - name: Test curl|bash installer
+        id: curl
+        run: |
+          set -u
+          url="https://raw.githubusercontent.com/metalbearco/mirrord/main/scripts/install.sh"
+          # Pipefail semantics: we check the pipeline with `if ! ... | bash;`
+          if ! curl -fsSL "$url" | bash >/tmp/install_sh.log 2>&1; then
+            echo "ok=false" >> "$GITHUB_OUTPUT"
+            echo "error=curl|bash installer failed from $url. See logs in this step." >> "$GITHUB_OUTPUT"
+            echo "----- installer output -----"
+            cat /tmp/install_sh.log || true
+            exit 0
+          fi
+
+          echo "ok=true" >> "$GITHUB_OUTPUT"
+          echo "error=" >> "$GITHUB_OUTPUT"
+          echo "curl|bash installer succeeded."
```

Comment:
Also here, let's verify mirrord is not installed before installing, then after installing let's verify `mirrord --version`